provider/core/src/resource.rs

```diff
@@ -425,9 +425,9 @@ impl From<&Locale> for ResourceOptions {
 }
 
 impl ResourceOptions {
-    pub fn cmp_bytes(&self, other: &[u8]) -> Ordering {
+    pub fn strict_cmp(&self, other: &[u8]) -> Ordering {
```

Comment:
Thought: I'm not sure if we want to bring the change into `ResourceOptions` as well, but I guess we probably do.